Packet++/src/DnsLayer.cpp

```diff
@@ -610,7 +611,7 @@ DnsQuery* DnsLayer::addQuery(const std::string& name, DnsType dnsType, DnsClass
 	}
 
 	// increase number of queries
-	getDnsHeader()->numberOfQuestions = htons(getQueryCount() + 1);
+	getDnsHeader()->numberOfQuestions = htons((uint16_t)getQueryCount() + 1);

```

Comment:
what is the reason for adding these casts? is it because on some platforms the size of `size_t` and `uint16_t` different?

Reply:
I tried to decrease the number of warnings in Visual Studio. I just forgot to remove this change.

Reply:
I will return back the old code